docs/superpowers/specs/2026-04-29-self-ref-chain-fixture-test-design.md

```
@@ -0,0 +1,124 @@
+| `file_6.R`–`file_8.R` | `func_N <- function(x) x + 1` + `source("file_N+1.R")` |
+| `file_9.R` | `func_9 <- function(x) x + 1` + `leaf_var <- 42` |
+
+`file_5.R` line layout (0-indexed):
+
+```text
+line 0: func_5 <- function(x) x + 1
+line 1: source("file_6.R")
+line 2: xyz <- xyz
+```
+
+At `xyz <- xyz` on line 2, the RHS `xyz` starts at **column 7** (after `xyz <- `). This is the single query point used for all scope assertions.
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)